cost/azure/unattached_volumes/azure_delete_unattached_volumes.pt

```diff
@@ -346,7 +342,7 @@ script "js_get_logactivity", type: "javascript" do
     var expectedDate=new Date();
     expectedDate.setDate(expectedDate.getDate()- parseInt(param_unattached_days));
     var date2=expectedDate.getFullYear()+'-'+(expectedDate.getMonth()+1)+'-'+expectedDate.getDate();
-    var filter= "eventTimestamp ge '"+date2+"T00:00:00Z' and eventTimestamp le '"+date1+"T23:59:59Z' and resourceProvider eq 'Microsoft.Compute' and resourceType eq 'Microsoft.Compute/VirtualMachines'"
+    var filter= "eventTimestamp ge '"+date2+"T00:00:00Z' and eventTimestamp le '"+date1+"T23:59:59Z' and eventChannels eq 'Admin, Operation' and resourceId eq '" + resourceID + "'"
```

Comment:
Levels should be included like:
and levels eq 'Critical,Error,Warning,Informational' at the very end of the filter query string.

Reply:
To use this, we would need to change line 355 to 'api-version': '2017-03-01-preview',